models/index.js

```diff
@@ -1,13 +1,19 @@
 const { User } = require('./User')
 const { Matcher } = require('./Matcher')
 const { Party } = require('./Party')
+const { UserParty } = require('./UserParty')
 
 // Need to refactor
-User.belongsToMany(Matcher, {through: Party})
-Matcher.belongsToMany(User, { through: Party})
+User.hasMany(Matcher); // User can have multiple Matchers
+Matcher.belongsTo(User); // Matcher belongs to one User
+
+User.belongsToMany(Party, { through: 'UserParty' }); // User can join several parties
```

Comment:
Correct me if I'm wrong, but I thought we had planned for a user to belong to one party at a time.